docs/source/how-to/customize-initialization.rst

```diff
@@ -93,6 +94,9 @@ Preloads can also be registered with configuration at the following values:
      nanny:
        preload: []
        preload_argv: []
+     client:
```

Comment:
This block should be one level to the left so that `distributed.scheduler` and `distributed.client` are on the same level. Right now it looks like the `client` section is inside the `worker` block

Reply:
I don't think that's correct? At least in my editor, the tab lines for `scheduler`, `worker`, `nanny`, and `client` are all at the same level:
<img width="229" alt="Screen Shot 2022-02-09 at 2 04 01 PM" src="https://user-images.githubusercontent.com/4396228/153271829-03b9a867-f5d1-4f81-bc9c-2a9df891274f.png">

Reply:
Whoops, you're totally right. That's what I get for inspecting indentation levels in a PR diff : )